components/layout/fragment.rs

```diff
@@ -4,7 +4,7 @@
 
 //! The `Fragment` type, which represents the leaves of the layout tree.
 
-#![deny(unsafe_code)]
+#![forbid(unsafe_code)]
```

Comment:
So far I've avoided `forbid` in layout just because it makes debugging annoying, and doesn't really add anything over `deny`, as any commit that added an `allow` of unsafe code could just remove the `forbid` (and highfive doesn't do anything different).